### PR TITLE
Updated admin based on mocks

### PR DESCRIPTION
**Pull request by tbaxter-18f**

### CODE REVIEW

**Comment by tbaxter-18f** [comment]
@cmajel here's at least a partial screenshot. Please let me know if you spot things you'd like changed before we merge. And of course we can always iterate, too.

Screen Shot 2021-04-08 at 5 05 55 PM

File: nrm_django/nrm_site/templates/admin/includes/fieldset.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    {% for line in fieldset %}
+        <div class="form-row{% if line.fields|length_is:'1' and line.errors %} errors{% endif %}{% if not line.has_visible_field %} hidden{% endif %}{% for field in line %}{% if field.field.name %} field-{{ field.field.name }}{% endif %}{% endfor %}">
+            {% if line.fields|length_is:'1' %}{{ line.errors }}{% endif %}
+            {% for field in line %}
+                <div{% if not line.fields|length_is:'1' %} class="fieldBox{% if field.field.name %} field-{{ field.field.name }}{% endif %}{% if not field.is_readonly and field.errors %} errors{% endif %}{% if field.field.is_hidden %} hidden{% endif %}"{% elif field.is_checkbox %} class="checkbox-row"{% endif %}>
+                    {% if not line.fields|length_is:'1' and not field.is_readonly %}{{ field.errors }}{% endif %}
+                    {% if field.is_checkbox %}
+                        {{ field.field }}{{ field.label_tag }}
+                    {% else %}
+                        {% if 'required' in field.label_tag %}
```
Comment (tbaxter-18f) [inline nrm_django/nrm_site/templates/admin/includes/fieldset.html:15]: This bit right here is the part we copied fieldset for.. so we could make the required fields more accessible.

File: nrm_django/grants/admin.py [hunk 2]
```diff
@@ -39,131 +39,157 @@ class GrantAdmin(admin.ModelAdmin):
...
-            },
-        ),
-        (
-            "Details",
-            {
-                "fields": (
-                    ("gid",),
-                    ("proj_desc", "areas_effected"),
+                    ("applicant_name",),
+                    # Org goes here
```
Comment (tbaxter-18f) [inline nrm_django/grants/admin.py:52]: We think org will come from contacts. Maybe.

File: nrm_django/grants/admin.py [hunk 3]
```diff
@@ -39,131 +39,157 @@ class GrantAdmin(admin.ModelAdmin):
...
+                    ("progrm_responsibility_type",),
+                    # program categories
```
Comment (tbaxter-18f) [inline nrm_django/grants/admin.py:54]: @neilmb , do you have any idea where this one might be tucked away? I'm not seeing it on grants, and per @cmajel 's research, it should be required.

File: nrm_django/grants/admin.py [hunk 4]
```diff
@@ -39,131 +39,157 @@ class GrantAdmin(admin.ModelAdmin):
...
+                    ("app_submit_date", "app_received_date"),
+                    # Financial assistance
```
Comment (tbaxter-18f) [inline nrm_django/grants/admin.py:56]: this one is a simple boolean in @cmajel 's mocks, so I don't think it's another table, but I also can't find a field it maps to. May be a Jamie question.

File: nrm_django/grants/admin.py (hunk 2 above)
Comment (neilmb) [reply nrm_django/grants/admin.py:52]: This is getting hooked up in https://github.com/18F/NRM-Grants-Agreements/tree/156-contacts

File: nrm_django/grants/admin.py (hunk 3 above)
Comment (neilmb) [reply nrm_django/grants/admin.py:54]: These are in a separate table: II_GA_IP_CATEGORIES that we will need to import and hook up.

File: nrm_django/grants/admin.py (hunk 4 above)
Comment (neilmb) [reply nrm_django/grants/admin.py:56]: I don't know, I think Jamie needs to answer that.

**Review by neilmb — APPROVED** [review]

File: nrm_django/grants/admin.py (hunk 3 above)
Comment (tbaxter-18f) [reply nrm_django/grants/admin.py:54]: Logged https://github.com/18F/NRM-Grants-Agreements/issues/182 to make that happen ASAP.

File: nrm_django/grants/admin.py (hunk 4 above)
Comment (tbaxter-18f) [reply nrm_django/grants/admin.py:56]: @cmajel can you ask about this when y'all are looking at the Figma mocks today?

**Pull request merged**